js/templates/notification.html

```diff
@@ -13,7 +13,7 @@
                             <%= polyglot.t('NotificationOrderConfirmed') %>
                         </div>
                         <div class="textSize11px floatLeft txt-fade"><%= ob.moment(new Date(ob.timestamp*1000)).locale(window.lang).format('MMM D, h:mm A') %></div>
-                    <% } else if(ob.type == "dispute_close") { %>
```

Comment:
Same concern as on the other PR. If indeed the server is inconsistently sending types with different case, then the bug should be on the server. IMHO, it's poor practice to complicate a client code base to account for fixable defects in the server.